audit/.support.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+. "$(dirname "${BASH_SOURCE[0]}")/../.common.sh"
+
+INSTANCE_ID=1
+
+AUTH_AUDIT_FILE="/srv/wrenam/auth/log/authentication.csv"
+
+read_audit_file() {
+  exec_am $INSTANCE_ID bash -c "cat $AUTH_AUDIT_FILE"
+}
+
+empty_audit_file() {
+  exec_am $INSTANCE_ID bash -c "echo "$AUTH_AUDIT_FILE_HEADER" > $AUTH_AUDIT_FILE"
```

Comment:
Not sure if this is safe to do. This depends on how the audit handler keeps file descriptors / references and how the filesystem works. So I am not sure you can safely empty file while it is being opened by another process.

Reply:
Ok, I can compare the file length before and after authenticating instead of completely emptying the file.